docusaurus.config.js

```diff
@@ -215,4 +215,10 @@ const config = {
         }),
 };
 
-module.exports = config;
+module.exports = {
+    ...config,
+    markdown: {
+        mermaid: true,
+      },
+      themes: ['@docusaurus/theme-mermaid'],
+    };
```

Comment:
EOL Is Missing, also @mhmohona Can we put the config for markdown and mermaid inside the existing config object instead of this way?

Reply:
umm let me check if that works.

Reply:
Any updates on this?

Reply:
nope :( I am not sure how to do it.

Reply:
We still have one problem to solve. How is it going? @mhmohona @adithyaakrishna

Reply:
Mermaid diagram renders perfectly and I didnt see any problem in the site in the way I put the code here.